client/modules/i18n/startup.js

```diff
@@ -65,10 +65,13 @@ Meteor.startup(() => {
         packageNamespaces.push(pkg.name);
       }
 
-
-      const shop = Shops.findOne({
+      let shop;
+      shop = Shops.findOne({
         _id: shopId
       });
+      if (!shop) {
+        shop = null;
```

Comment:
This check sets `shop` to `null`. Then immediately below it, we're referencing `shop.language`. That will throw an error. 

And that's what I got immediately I signed out: `TypeError: Cannot read property 'language' of null`